packages/client/bin/cli.ts

```diff
@@ -14,7 +14,7 @@ import { LevelDB } from '../src/execution/level.ts'
 import { generateVKTStateRoot } from '../src/util/vkt.ts'
 
 import { helpRPC, startRPCServers } from './startRPC.ts'
-import { generateClientConfig, getArgs } from './utils.ts'
+import { generateClientConfig, generateRpcConfigs, getArgs } from './utils.ts'
```

Comment:
`generateRpcConfigs` -> `generateRpcConfig` (so: without the `s`)

to align with `generateClientConfig`

Reply:
(also further down the line there are a few `Configs`, can you please also rename consistently? Thanks! (we are a bit picky on naming things 🙂 ))